projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -3234,7 +3234,10 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         )
             .subscribe(() => {
                 this.zone.run(() => {
-                    this.notifyChanges(true);
+                    // do not trigger reflow if element is detached.
+                    if (this.document.contains(this.nativeElement)) {
```

Comment:
if you can you should try the fix on iPad/Safari before merging as  I remember Safari having some weird shortcomings with this API